### DISCUSSION TITLE
Rust UI

### DISCUSSION THREAD

**Discussion by mezeipetister** [fzyzcjy/flutter_rust_bridge#2232] [General]
Hi guys. Could you help me where to find docs about frb(ui_state) and frb(ui_mutation)? Also could you help me how to generate dart code in rust_ui_counter example project? 

Thank you in advance.

**Comment by fzyzcjy** [comment]
Oh it is not there, but I will write it later.

> Also could you help me how to generate dart code in rust_ui_counter example project?

As is shown in https://cjycode.com/flutter_rust_bridge/guides/miscellaneous/unreleased, try:

![image](https://github.com/user-attachments/assets/63ab2324-008d-4c05-a984-92b29d0855f6)
